docs/attributes-registry/gen-ai.md

```diff
@@ -0,0 +1,48 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Large Language Model (LLM)
+
+<!-- toc -->
+
+- [Generic LLM Attributes](#generic-llm-attributes)
+  - [Request Attributes](#request-attributes)
+  - [Response Attributes](#response-attributes)
+  - [Event Attributes](#event-attributes)
+
+<!-- tocstop -->
+
+## Generic LLM Attributes
+
+### Request Attributes
+
+<!-- semconv registry.llm(omit_requirement_level,tag=llm-generic-request) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `gen_ai.llm.request.max_tokens` | int | The maximum number of tokens the LLM generates for a request. | `100` |
```

Comment:
do you believe we'll ever have common fields between LLM and non-LLM GenAI?

In other words, what would break if we had a root `llm` namespace and an imaginary `image_gen_ai` root namespace in the future?